common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -389,6 +389,7 @@ enum eFreqType {
     FREQ_5G     = 1,
     FREQ_58G    = 2,
     FREQ_24G_5G = 3,
+    FREQ_60G    = 4,
```

Comment:
This PR doesn't end up using it, so why add it?

Reply:
removed